util.js

```diff
@@ -74,7 +74,7 @@ function trimSlashes (str) {
 }
 
 function cachedPrebuild (url) {
-  const digest = crypto.createHash('md5').update(url).digest('hex').slice(0, 6)
+  const digest = crypto.createHash('sha1').update(url).digest('hex').slice(0, 6)
```

Comment:
I'd probably go with `sha512` here as that's what (p)npm use for content-based hashing.

Reply:
done